src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -118,6 +119,8 @@
 
 	private final PlayerSpigotMock playerSpigotMock = new PlayerSpigotMock();
 	private final List<AudioExperience> heardSounds = new LinkedList<>();
+	private final Map<UUID, List<Plugin>> hiddenPlayers = new HashMap<>();
+	private final List<UUID> hiddenPlayersDeprecated = new ArrayList<>();
```

Comment:
To be honest I am not the biggest fan of this being split into two here but I guess it is fine for now...

Reply:
It can be done by making a fake plugin `private static final Plugin fakeDeprecatedPlugin = new ???()` and then just using this plugin on this deprecated functions like that `hidePlayer(fakeDeprecatedPlugin, player);` `showPlayer(fakeDeprecatedPlugin, player);` but I don't know how to get easy and safe this fake plugin object without just implementing like 20 functions from this interface

Reply:
We could also just have an "empty list" represent this but that would probably be a bit more work than necessary. It is fine for now